css/properties/interpolate-size.json

```
@@ -0,0 +1,39 @@
+{
+  "css": {
+    "properties": {
+      "interpolate-size": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-values-5/#interpolate-size",
+          "support": {
+            "chrome": {
+              "version_added": "128"
```

Looks like the ChromeStatus page says 129:

```suggestion
              "version_added": "129"
```